uxarray/conventions/ugrid.py

```diff
@@ -0,0 +1,323 @@
+from uxarray.constants import INT_FILL_VALUE, INT_DTYPE
+
+CONVENTIONS_ATTR = "UGRID-v1.0"
+
+# minimum grid topology, additions are made depending on what is present in a grid
+BASE_GRID_TOPOLOGY_ATTRS = {
+    "cf_role": "mesh_topology",
+    "topology_dimension": 2,
+    # dimensions
+    "face_dimension": "n_face",
+    "node_dimension": "n_node",
+    # coordinates
+    "node_coordinates": "node_lon node_lat",
+    # connectivity
+    "face_node_connectivity": "face_node_connectivity",
+}
+
+DIM_NAMES = [
+    "n_node",
+    "n_edge",
+    "n_face",
+    "n_max_face_nodes",
+    "n_max_face_edges",
+    "n_max_face_faces",
+    "n_max_edge_nodes",
+    "n_max_edge_edges",
+    "n_max_edge_faces",
+    "n_max_node_faces",
+    "n_max_node_edges",
+    "two",
+]
+
+NODE_DIM = "n_node"
+EDGE_DIM = "n_edge"
+FACE_DIM = "n_face"
+N_MAX_FACE_NODES_DIM = "n_max_face_nodes"
+
+NODE_COORDINATES = ["node_lon", "node_lat"]
+
+# coordinates (spherical)
+NODE_LON_ATTRS = {
+    "standard_name": "longitude",
+    "long name": "Longitude of the corner nodes of each face",
+    "units": "degrees_east",
+}
+
+NODE_LAT_ATTRS = {
+    "standard_name": "latitude",
+    "long name": "Latitude of the corner nodes of each face",
+    "units": "degrees_north",
+}
+
+EDGE_COORDINATES = ["edge_lon", "edge_lat"]
+
+EDGE_LON_ATTRS = {
+    "standard_name": "longitude",
+    "long name": "Longitude of the center of each edge",
+    "units": "degrees_east",
+}
+
+EDGE_LAT_ATTRS = {
+    "standard_name": "latitude",
+    "long name": "Latitude of the center of each edge",
+    "units": "degrees_north",
+}
+
+FACE_COORDINATES = ["face_lon", "face_lat"]
+
+FACE_LON_ATTRS = {
+    "standard_name": "longitude",
+    "long name": "Longitude of the center of each face",
+    "units": "degrees_east",
+}
+
+FACE_LAT_ATTRS = {
+    "standard_name": "latitude",
+    "long name": "Latitude of the center of each face",
+    "units": "degrees_north",
+}
+
+# coordinates (cartesian)
+NODE_X_ATTRS = {
+    "standard_name": "x",
+    "long name": "x location of the corner nodes of each face ",
+    "units": "meters",
+}
+
+NODE_Y_ATTRS = {
```

Comment:
Does these X, Y,Z all refer to the cartesian coordinates? We better specify that in the long name

Reply:
They do. This is a good idea.